### ISSUE TITLE
The command report can (still) be used to meta the roundtype

### ISSUE THREAD

**Issue by BeeSting12** [tgstation/tgstation#33556]
Sample Command Report (Actual round type was ops)

Central Command Status Summary
--------------------------------------------------------------------------------
Central Command has intercepted and partially decoded a Syndicate transmission with vital information regarding their movements. The following report outlines the most likely threats to appear in your sector.
--------------------------------------------------------------------------------
The transmission mostly failed to mention your sector. It is possible that there is nothing in the Syndicate that could threaten your station during this shift.
--------------------------------------------------------------------------------
Infernal creatures have been seen nearby offering great boons in exchange for souls. This is considered theft against Nanotrasen, as all employment contracts contain a lien on the employee's soul. If anyone sells their soul in error, contact an attorney to overrule the sale. Be warned that if the devil purchases enough souls, a gateway to hell may open.
--------------------------------------------------------------------------------
One of Central Command's trading routes was recently disrupted by a raid carried out by the Gorlex Marauders. They seemed to only be after one ship - a highly-sensitive transport containing a nuclear fission explosive, although it is useless without the proper code and authorization disk. While the code was likely found in minutes, the only disk that can activate this explosive is on your station. Ensure that it is protected at all times, and remain alert for possible intruders.
--------------------------------------------------------------------------------
Reports of an ancient banana blight outbreak that turn humans into monkeys has been reported in your quadrant. Any such infections may be treated with banana juice. If an outbreak occurs, ensure the station is quarantined to prevent a largescale outbreak at CentCom.

so, tldr: 
Possibility 1: Extended.
Possibility 2: Devil (not even in rotation)
Possibility 3: Nuke ops
Possibility 4: Monkey (also not in rotation)

So basically they have it narrowed down to two round types. This can happen with other modes- See IAA, Devil, meteor, and cult? It's cult! 

Here's my recommended solutions:
1. Change it back to the way it was before: Utter gibberish, totally useless.
2. Make it list x amount of possible round types and then the real one. Not entirely useless since you can rule out stuff but still better.
3. Make it show possible round types- The gamemode doesn't have to be in there.

**Comment by Shapsy** [comment]
https://i.imgur.com/ubdbXbu.png

This is not having the intended effect, people screaming out the possible roundtypes at roundstart is incredibly un-conducive to RP or fun. It's better random.

**Comment by Robustin** [comment]
Seems like a config issue - still haven't gotten a straight answer why the probability check isn't working properly.

**Comment by praisenarsie** [comment]
seems like a player issue more than anything, admins should just punish by adding more antags tbh

**Comment by lordpidey** [comment]
Currently disabled modes SHOULD be showing up much less than more common modes in the reports.  I'll do some checking to see if that's the case, if it's simply the code selecting things wrongly, then that's an easy fix.

But that's the thing about randomness, you're occasionally going to get reports like that.

**Comment by Robustin** [comment]
While it isn't ideal, the change I made where the current mode is no longer guaranteed to show up has resolved this issue.

**Comment by optimumtact** [comment]
ha ha ha ha ha

**Comment by ShizCalev** [comment]
sounds good enough to me.
